src/components/Vizmapper/VisualPropertyRender/Color.tsx

```diff
@@ -19,7 +19,171 @@ export function ColorPicker(props: {
+    ['#f7fdf1', '#e0f6de', '#caefca', '#a2e4bb', '#69d6cb', '#0bbfdb', '#009ac9', '#0065ac'],
+    ['#fff8fc', '#f1e5f3', '#d6d7ea', '#aac6e1', '#aac6e1', '#009eca', '#009096', '#00735a'],
+    ['#fff8f4', '#ffe2e1', '#ffc8c6', '#ffa2bd', '#ff62ac', '#f700a4', '#ca008c', '#940085'],
+    ['#fff8ee', '#ffeacc', '#ffd7a3', '#ffbf89', '#ff8f5c', '#ff614a', '#f10011', '#b40000'],
+    ['#f7fdfd', '#e1eff6', '#c2d9ea', '#a1c5e1', '#95a1cf', '#9e73bd', '#9f41aa', '#860079'],
+    ['#ffffe7', '#fff8bf', '#ffe594', '#ffc847', '#ff9c00', '#ff6f00', '#e44500', '#a52300'],
+    ['#ffffcf', '#ffefa4', '#ffe594', '#ffb646', '#ff8f33', '#ff3b1e', '#fd000a', '#cc0026'],
+    ['#fff6f2', '#ffe2d6', '#ffbfa8', '#ff9478', '#ff654c', '#ff1126', '#e60012', '#b40000'],
+    ['#f9f5fa', '#ebe4f2', '#dfbfe0', '#da9ad0', '#f662bb', '#ff0097', '#e90060', '#ac0048'],
+    ['#fff6ed', '#ffe8d2', '#ffd3a8', '#ffb26e', '#ff8f33', '#ff6600', '#f13b00', '#a52300'],
+    ['#f7fcff', '#dfeef9', '#c8e1f2', '#9cd2e6', '#5fbade', '#19a0d0', '#007fc1', '#0050a2'],
+    ['#fdfcfd', '#f2eff7', '#dfdfef', '#c4c4e2', '#a9a4d1', '#8d88c5', '#7b59b0', '#5d0095'],
+    ['#ffffe7', '#f7fdbc', '#d8f4a7', '#a8e493', '#66d17f', '#00b962', '#009347', '#006836'],
+    ['#ffffff', '#f2f2f2', '#dedede', '#c5c5c5', '#a0a0a0', '#7e7e7e', '#5c5c5c', '#2a2a2a'],
+    ['#f7fdfd', '#e5f7fa', '#cbf0e9', '#91e0cf', '#47cdad', '#00bb7e', '#009a49', '#006623'],
+    ['#ffffdb', '#ecfab5', '#c5eeb9', '#6fd7c2', '#00c3cd', '#009fca', '#006bb5', '#003293'],
+    ['#fff8fc', '#f0eaf4', '#d6d7ea', '#aac6e1', '#6fb5d7', '#009eca', '#007ebc', '#005a89'],
+  ];
+
+  return (
+    <Box>
+      <SwatchesPicker
+        width={1000}
+        colors={customColors}
+        color={localColorValue}
+        onChange={(color: any) => {
+          setLocalColorValue(color.hex)
+          debouncedValueChange(color.hex)
+        }}
+      />
+    </Box>
+  )
+}
+
+export function ColorPickerDiverging(props: {
+  currentValue: ColorType | null
+  onValueChange: (color: ColorType) => void
+}): React.ReactElement {
+  const { onValueChange, currentValue } = props
+  const debouncedValueChange = debounce(onValueChange, 200)
+
+  // use local state to appear instantaneous in the color picker,
+  // but the actual visual style model updates are debounced
+  const [localColorValue, setLocalColorValue] = React.useState<ColorType>(
+    currentValue ?? `#ffffff`,
+  )
+
+  const customColors = [
+    ['#ff0000', '#ff304d', '#ff939b', '#ffdddf', '#e0dfff', '#9e9cff', '#5750ff', '#1900ff'],
+    ['#cd002c', '#ed5e52', '#ffa989', '#ffdecc', '#d2e9f3', '#8ecee4', '#1ba1cd', '#0073b9'],
+    ['#ca5a00', '#f38600', '#ffbc64', '#ffe2bb', '#dddfef', '#bcb3da', '#8f7db8', '#682497'],
+    ['#8e2191', '#ac78b7', '#cfacd7', '#eed8ec', '#d8f3d7', '#a0e2a6', '#3bbb66', '#008739'],
+    ['#ee2957', '#ff6a43', '#ffb262', '#ffe38e', '#e5f89b', '#a6e4aa', '#47cdae', '#0096c8'],
```

Comment:
These should be defined in the colors model impl so that they can be reused in other places in the app that needs to deal with colors.